### PR TITLE
Add audio normalization device config option

### PR DESCRIPTION
**Pull request by jsbmg**
- Add `normalization` device config option for audio normalization
- Update device default config

### CODE REVIEW

File: spotify_player/src/streaming.rs [hunk 1]
```diff
@@ -174,6 +174,11 @@ pub fn new_connection(
             .to_string()
             .parse::<Bitrate>()
             .unwrap_or_default(),
+	normalisation: device
+	    .normalization
+	    .to_string()
+	    .parse::<bool>()
+	    .unwrap_or_default(),
```
Comment (aome510) [inline spotify_player/src/streaming.rs:181]: the field itself is `bool` already. This part is redundant.

```suggestion
	    .normalization,
```

**Review by aome510 — CHANGES_REQUESTED** [review]
Thanks for the PR. Can you update https://github.com/aome510/spotify-player/blob/master/examples/app.toml as well?

**Comment by jsbmg** [comment]
Thanks for the feedback. Just updated with those two changes.

**Review by aome510 — APPROVED** [review]

**Comment by aome510** [comment]
@jsbmg can you fix the CI failure?

**Comment by jsbmg** [comment]
> @jsbmg can you fix the CI failure?

Hey, sorry, had a busy week, but thanks for fixing that and merging it. Appreciate your work.

**Pull request merged**